web/app/types/document.d.ts

```diff
@@ -12,8 +14,16 @@ export interface HermesDocument {
   docNumber: string;
   docType: string;
   title: string;
+  locked?: boolean;
+  owners?: string[];
+  appCreated?: boolean;
+  contributors?: HermesUser[];
+  approvers?: HermesUser[];
+  changesRequestedBy?: unknown[]; // TODO: Define this
```

Comment:
This should also be `HermesUser[]`

Reply:
Actually I'm inferring that it's a `string[]` like `approvedBy`:

```
  get hasRequestedChanges() {
    return this.args.document.changesRequestedBy?.includes(
      this.args.profile.email
    );
  }
```